### PR TITLE
Docs for IDP session extender endpoint added

### PR DESCRIPTION
**Pull request by ashendes**

### CODE REVIEW

File: en/docs/develop/idp-session-extender-endpoint.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Identity Provider Session Extender Endpoint
+
+!!! info
+    To access this endpoint, use the following URL.
+    `https://<IS_HOST>:<IS_PORT>/identity/extend-session`
+
+This endpoint can be used to extend the user's session at the Identity Provider. In order to identify the session that
```
Comment (sherenewso2) [inline en/docs/develop/idp-session-extender-endpoint.md:7]:
```suggestion
This endpoint can be used to extend the user's session at the identity provider. To identify the session that
```

File: en/docs/develop/idp-session-extender-endpoint.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+``` toml
+[session_extending_endpoint]
+enabled = true
+```
+        
+### Extracting the session identifier value
+
+If the above config has been enabled, the session identifier can be extracted from the ID token. 
+The ID token should be decoded, and the value of the claim `isk`, which is the identifier of the session, should be
+ read.
```
Comment (sherenewso2) [inline en/docs/develop/idp-session-extender-endpoint.md:29]:
```suggestion
To do this, decode the ID token and read the value of the claim `isk`, which is the identifier of the session.
```

File: en/docs/develop/idp-session-extender-endpoint.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ 
+### Supported requests
+
+##### Sending session identifier as a request parameter
+
+```java
+curl --location --request GET 'https://<IS_HOST>:<IS_PORT>/identity/extend-session?idpSessionKey=<SESSION_IDENTIFIER>'
+```
+
+`<SESSION_IDENTIFIER>` should be replaced with the `isk` value obtained from the ID token. This method is ideal
```
Comment (sherenewso2) [inline en/docs/develop/idp-session-extender-endpoint.md:39]:
```suggestion
Replace the `<SESSION_IDENTIFIER>` with the `isk` value obtained from the ID token. This method is ideal
```

File: en/docs/develop/idp-session-extender-endpoint.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ in cases where the use of third party cookies has been blocked.
+
+##### Sending session identifier as a cookie
+
+```java
+curl --location --request GET 'https://<IS_HOST>:<IS_PORT>/identity/extend-session' \
+--header 'Cookie: <SESSION_IDENTIFIER>'
+```
+
+In this case, `<SESSION_IDENTIFIER>` should be replaced with the `commonAuthId` cookie.
```
Comment (sherenewso2) [inline en/docs/develop/idp-session-extender-endpoint.md:49]:
```suggestion
In this case, replace the `<SESSION_IDENTIFIER>` with the `commonAuthId` cookie.
```

**Review by sherenewso2 — APPROVED** [review]

**Comment by sherenewso2** [comment]
Since this is a new doc, it needs to be included in the mkdocs.yml file to include it in the left navigation

**Comment by ashendes** [comment]
> Since this is a new doc, it needs to be included in the mkdocs.yml file to include it in the left navigation

Added to `mkdocs.yml` in 790d3db61417b1e62983ff6656fac9b9cb5b1e51

**Review by sherenewso2 — APPROVED** [review]

**Pull request merged**